src/iop/opposed.c

```diff
@@ -358,14 +357,15 @@ static float *_process_opposed(struct dt_iop_module_t *self, dt_dev_pixelpipe_io
       }
     }
     for_each_channel(c)
-      chrominance[c] = cr_sum[c] / fmax(1.0, cr_cnt[c]);    
+      chrominance[c] = cr_sum[c] / fmax(1.0, cr_cnt[c]);
```

Comment:
we can probably gain some speed here:

`fmaxf(1.0f, cr_cnt[c]);`

Reply:
Thanks for hinting...